pycalphad/core/custom_autowrap.py

```diff
@@ -276,6 +288,137 @@ def dump_h(self, routines, f, prefix, header=True, empty=True):
 
     dump_fns = [CCodeGen.dump_c, dump_h]
 
+    def routine(self, name, expr, argument_sequence, global_vars):
```

Comment:
Would it be possible to add some commentary on why we need to override this and what the major changes are compared to the CodeGen base class?

Reply:
Added. For reference: http://linediff.com/?id=5a3be237687f4b2d2a8b4567